docs/hellonode.md

```diff
@@ -126,6 +126,25 @@ Create a cluster via the Console: *Compute > Container Engine > Container Cluste
 
 It’s now time to deploy your own containerized application to the Kubernetes cluster! Please ensure that you have [configured](https://cloud.google.com/container-engine/docs/before-you-begin#optional_set_gcloud_defaults) `kubectl` to use the cluster you just created.
 
+**The rest of this document requires both the kubernetes client and server version to be 1.2.**  This should be true by default for new Google Container Engine clusters.
+
+```shell
+$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.0", GitCommit:"5cb86ee022267586db386f62781338b0483733b3", GitTreeState:"clean"}
+Server Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.0", GitCommit:"5cb86ee022267586db386f62781338b0483733b3", GitTreeState:"clean"}
+```
+
+**If either the client or server version is less than 1.2, see the [1.1 version of this document](https://github.com/kubernetes/kubernetes.github.io/blob/release-1.1/docs/hellonode.md).**
```

Comment:
Does that include the first line that says v1.2 is required?  Can we expect users to come to the site on either client or server v1.1.  If so are we responsible for letting them know that this will not work?  I guess since the guide tells the user to setup the cluster on GKE we can expect they are not already running one.